java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -2807,27 +2851,55 @@ public static ColumnVector fromBytes(byte... values) {
     return build(DType.INT8, values.length, (b) -> b.appendArray(values));
   }
 
+  /**
+   * Create a new vector from the given values.
+   */
+  public static ColumnVector fromUnsignedBytes(byte... values) {
```

Comment:
It would be good to have a comment on all of these that it treats the bits as if they were unsigned, like a reinterpret cast in c++, or something like that.

Reply:
Added a comment explaining the bits will be interpreted as unsigned for all the "from*" methods in `HostColumnVector` and `ColumnVector`.  Updated the docs of the ".asUnsigned*" methods to point to the `toUnsigned*` methods needed to properly interpret the data in Java.